internal/template.go

```diff
@@ -115,7 +115,10 @@ func handleOptions(options []string) (map[string]string, error) {
 	return opts, nil
 }
 
-func newAWSSession() *session.Session {
+func newAWSSession(profile string) *session.Session {
+	if profile != "" {
+		os.Setenv("AWS_PROFILE", profile)
```

Comment:
Instead of exporting the env variable you can add the profile to the `session.Options`.
It looks a bit safer than doing this 🤔 

// Specify profile for config and region for requests
sess, err := session.NewSessionWithOptions(session.Options{
    Profile: "profile_name",
})

Reply:
You're right. I will change it.